crates/uv/src/commands/project/run.rs

```diff
@@ -104,6 +106,41 @@ pub(crate) async fn run(
     // Initialize any shared state.
     let state = SharedState::default();
 
+    // Initialize env file, if necessary.
+    if !no_env_file {
+        let env_file_path = env_file.clone().unwrap_or(Path::new(".env").to_path_buf());
```

Comment:
You can use `unwrap_or_else` to avoid allocating (`.to_path_buf()`) in the event that `env_file` is provided.

Reply:
Good point! Thanks!